tests/acceptance/asset-load-test.js

```diff
@@ -1,6 +1,30 @@
+import Ember from 'ember';
 import { test } from 'qunit';
 import moduleForAcceptance from '../../tests/helpers/module-for-acceptance';
 
+const { run } = Ember;
+const { Promise } = Ember.RSVP;
+
+function waitFor(checkerFn, timeout = 1000) {
+  let start = Date.now();
+
+  return new Promise((resolve, reject) => {
+    let watcher = setInterval(() => {
+      if (!checkerFn()) {
+        return;
+      }
+
+      if (Date.now() - start > timeout) {
```

Comment:
Shouldn't this go before the other `if` statement? Otherwise, if `checkerFn` only ever returns false, we'll never actually reject.